cypress/support/commands.ts

```diff
@@ -0,0 +1,73 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add("login", (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+declare namespace Cypress {
+  interface Chainable<Subject> {
+    pageHeaderShouldBeVisible(headerText: string): void;
+    overviewTableRendersCorrectly(): void;
+    checkInPageNavigationLinks(): void;
+  }
+}
+
+Cypress.Commands.add('pageHeaderShouldBeVisible', (headerText) => {
+  cy.contains('h1', headerText).should('be.visible');
```

Comment:
I worry a little that this might get annoying as it's tied to content which might change, meaning you have to change the test. We should keep an eye on it